test.sh

```diff
@@ -2,8 +2,14 @@
 
 set -eux
 
+EXTRA_ARGS=""
+
+if [ -n "${TARGET-}" ]; then
+    EXTRA_ARGS="--release --target=${TARGET}"
```

Comment:
Can we also enable debug-assertions for release mode?

Reply:
I wonder, should I go for `RUSTFLAGS` only for cross-tests or do this unconditionally via `profile.release`?

Reply:
Done by adjust `RUSTFLAGS` so that actual release builds in production are not affected.